scss/_modal.scss

```diff
@@ -236,3 +236,26 @@
 @include media-breakpoint-up(xl) {
   .modal-xl { max-width: $modal-xl; }
 }
+
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-down($breakpoint) {
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
+
+    .modal#{$infix}-fullscreen {
+      width: 100vw;
+      max-width: inherit;
+      height: 100vh;
+      margin: 0;
+
+      .modal-content {
+        height: 100%;
+        border: 0;
+        @include border-radius(0);
+      }
+
+      .modal-body {
+        overflow-y: auto;
```

Comment:
I think we can safely move this to the global `.modal-body`, would generate less code.